lib/chef-dk/policyfile/chef_server_cookbook_source.rb

```diff
@@ -47,8 +60,19 @@ def desc
         "chef_server(#{uri})"
       end
 
+      private
+
+      def http_connection_for(base_url)
+        @http_connections[base_url] ||= Chef::HTTP::Simple.new(base_url)
```

Comment:
Wouldn't we need authentication? If so, I think you can just use this: https://github.com/chef/chef-dk/blob/master/lib/chef-dk/authenticated_http.rb